src/configuration.ts

```diff
@@ -19,6 +19,13 @@ import * as fs from "fs";
 import * as path from "path";
 import * as findup from "findup-sync";
 
+import {arrayify, objectify} from "./utils";
+
+export interface IConfigFile {
```

Comment:
So just leave this as is, right? Exported things are a blurry line between private/public, but I assume we're only treating as public things in the top level package